### ISSUE TITLE
Update QGreenland ADC portal to use cesium

### ISSUE THREAD

**Issue by trey-stafford** [QGreenland-Net/.github#30]

**Comment by rushirajnenuji** [comment]
Related [MetacatUI issue](https://github.com/NCEAS/metacatui/issues/2375)

**Comment by rushirajnenuji** [comment]
Update: 
- The QGreenland portal has been cloned from production to the dev environment for testing, and is now open to DataONE wide datasets (all of DataONE repositories with QGreenland data)
- Currently WIP: running PDG pipeling - i.e. running PDG packages `viz-staging` and `viz-raster` on one of the QGreenland datasets. 
- I'm using the `Hydrology > Ice basins` layer from the QGreenland package as an example to generate the web tiles. 
    - Ran into some issues with low disk space on my machine. The processing occupied 250+ gigs on my machine
    - Had to re-run some of the processing.
    - Juliet and Robyn from the ADC / PDG team have been answering my questions and helping me get familiar with the PDG workflow
    - Still waiting for the workflow to complete the processing.
- I've set up local Cesium for testing this web-tiles once this processing is complete.

Next steps:
- make these output tiles web accessible
- update some of the portal configuration to  start reading these tiles
- test and explore the data on the Cesium
- pass the portal to the team for testing.

**Comment by mfisher87** [comment]
>Ran into some issues with low disk space on my machine. The processing occupied 250+ gigs on my machine

Whoa! Did you identify the cause?

**Comment by mfisher87** [comment]
First pass completed, will open another ticket for ongoing work
